src/dst/bind.cpp

```diff
@@ -1148,6 +1194,26 @@ static std::unique_ptr<Expr> fracture(std::unique_ptr<Top> top) {
     }
   }
 
+  // Any *import* that is only used once must be unused
+  // as that singular use came from the export.
+  for (const auto &imp : imports) {
+    auto it = gbinding.index.find(imp.first);
+    if (it == gbinding.index.end()) {
+      continue;
+    }
+
+    auto &i = gbinding.defs[it->second];
+
+    // Generated defs shouldn't be included
+    if (i.expr->flags & FLAG_SYNTHETIC) {
+      continue;
+    }
+
+    if (i.uses == 1) {
```

Comment:
Is the reason `uses` has 1 and not zero because the import "uses" it? If so I think a comment should be added here explaining that.

Reply:
Hmm, I though I had left a comment already, it is `1` here because anything _exported_ (which is everything imported) gets it usage incremented by 1 by virtue of being exported.

 Ignoring this PR,  if we don't increment all exports by 1 then any unused export triggers a warning